src/app/app.component.html

```diff
@@ -6,10 +6,28 @@
     </mat-grid-list>
   </mat-toolbar>
   <mat-grid-list cols="3" rowHeight="50px" gutterSize="5px">
-    <mat-grid-tile routerLink="/dashboard">Dashboard</mat-grid-tile>
-    <mat-grid-tile routerLink="/userlist">User List</mat-grid-tile>
-    <mat-grid-tile routerLink="/positive-case">Positive Cases</mat-grid-tile>
-    <mat-grid-tile routerLink="/AssignedCases">Assigned Cases</mat-grid-tile>
+    <div [hidden]="!login.isAuthenticated()">
+      <mat-grid-tile routerLink="/dashboard">Dashboard</mat-grid-tile>
+    </div>
+    
+    <div [hidden]="!login.isAuthenticated() || login.roleIsPerson() || login.roleIsUnspecified()">
+      <mat-grid-tile routerLink="/userlist">User List</mat-grid-tile>
```

Comment:
You mentioned that you wanted to change the column name from User List in the frontend, you would just need to change it here

Reply:
Done!